common_types/src/schema.rs

```diff
@@ -745,6 +750,21 @@ impl Schema {
     pub fn string_buffer_offset(&self) -> usize {
         self.column_schemas.string_buffer_offset
     }
+
+    /// Data format in storage
+    pub fn storage_format(&self) -> StorageFormat {
+        // TODO: parse it from table options
+        match std::env::var("CERESDB_TABLE_FORMAT") {
```

Comment:
parse from table option is more elegant. If we choose to use env we should document it.

Reply:
Temporary is acceptable, but we need to doc this env someplace like PR's desc.